components/shared/Pagination.tsx

```diff
@@ -0,0 +1,22 @@
+import { styled } from "@mui/material/styles";
```

Comment:
Is it possible to make MUI work with aphrodite? 

Since we have aphrodite everywhere else in the app, it would be nice to keep the same paradigm

Reply:
@lightninglu10 I don't think so. It works with emotion or cssjs out of the box. 

@manveerxyz can you run yarn build locally on master and your branch to see if including MUI here impacts the size of the page significantly. Historically we tried to limit MUI to just the ref manager since it has a heavy footprint. If the footprint is not too bad, that's okay

Reply:
@yattias `/hubs` is `8KB` on master, and `12.3KB` on my branch.

Master:
```
├ λ /footer                                                1.52 kB         626 kB
├ λ /health                                                422 B           625 kB
├ ● /hubs (ISR: 600 Seconds) (1214 ms)                     8 kB            633 kB
├ ● /hubs/[slug]                                           515 B           683 kB
├ ● /hubs/[slug]/live                                      519 B           683 kB
```

Hub-Pagination:
```
├ λ /footer                                          1.52 kB         626 kB
├ λ /health                                          422 B           625 kB
├ ● /hubs (ISR: 600 Seconds) (6012 ms)               12.3 kB         661 kB
├ ● /hubs/[slug]                                     515 B           683 kB
├ ● /hubs/[slug]/live                                519 B           683 kB
```

Reply:
@lightninglu10 I tried using aphrodite but it doesn't work. I used `styled` since we use that in one other place in the codebase to style an MUI component.

Kobe is right that it works with Emotion or classic CSS. The other alternative is to hardcode `styles` in the component, but I'd prefer using the `styled` function from MUI since it seems more standard for just tweaking styling.